src/pudl/metadata/resources/gridpathratk.py

```diff
@@ -0,0 +1,29 @@
+"""Table definitions for data coming from the GridPath Resource Adequacy Toolkit."""
+
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_gridpathratk__aggregated_extended_profiles": {
```

Comment:
I don't have an opinion here, guessing @cmgosnell might.

Reply:
But, I don't think this matches our existing naming conventions. There should be a `yearly` if the data is yearly, or some other indication of data frequency.

Reply:
Ah yes, oops. Should be hourly.

Reply:
+1 to adding a frequency. I also think that `gridpathratk` as the dataset name doesn't conform to the rest of our data sources. I know that `gp` is less recognizable than `eia` in our little acronym corner but in this case I think `gp` is the "provider" and perhaps `ratk` is the form.

agreed that its hard to think about the main table-y part of the name... these are definitely going to be generation profiles. are all of the possible `ratk` tables "extended" in dates? if so I would probably take that out because its less what it is. but its not a hill i will die on

Reply:
@cmgosnell Can you say where you're thinking this distinction between the provider and the "form" needs to be made?  Here `core_gridpathratk` would be analogous to e.g. `core_eia860` which includes both the agency and the form.

Reply:
I'm speaking specifically of the `gridpathratk` part of this name. I'm mostly nothing that all of our other data sources are acronyms of "agencies" and then numbers or acronyms of "forms" and this one is  `gridpath` (agency) and `ratk` (form).

I think `gprat` is more in line. Or idk the GP model is actually published by GridLab... Should it be `gl` and the `gpra` or `gprat`?

Reply:
I feel like we can get away with using short acronyms for the familiar agencies like `ferc`, `eia`, and `epa` because in this context it's clear what they refer to, but `gp` or `gl` doesn't communicate who/where the data is coming from.

I'm pretty sure the GridPath model is independently developed by Ana / Blue Marble Analytics, and the GridPath Resource Adequacy Toolkit was a study supported by GridLab which used the GridPath model. I think the data was largely compiled by Elaine / Moment Energy Insights. So I don't know that there's a single clear "agency" like source to refer to.  I also suspect that this will be a one-off. Maybe just using `gridpathratoolkit` which is how they refer to it everywhere would be clearest.

> The GridPath RA Toolkit refers to both the datasets and the algorithms that were developed to support the 2026 Western US case study, which can be downloaded from this site.